python/pyarrow/array.pxi

```diff
@@ -2888,23 +2888,6 @@ cdef class ExtensionArray(Array):
         # otherwise convert the storage array with the base implementation
         return Array._to_pandas(self.storage, options, **kwargs)
 
-    def to_numpy(self, **kwargs):
-        """
-        Convert extension array to a numpy ndarray.
-
-        This method simply delegates to the underlying storage array.
```

Comment:
Can you move this sentence to the docstring of the to_numpy method on the base class ("For this extension arrays, this method ...")